maint/gdb/gdbhelpers

```diff
@@ -1,7 +1,7 @@
 # Copyright 2018 Red Hat, Inc.
 # Author: Jan Pokorny <jpokorny@redhat.com>
 # Part of pacemaker project
-# SPDX-License-Identifier: GPL-2.0-or-later
+# SPDX-License-Identifier: FSFAP
```

Comment:
Re bureaucracy:
only on opt-in basis, if they want to ship that file in the end-use
packages, but I guess they don't (for ad-hoc debugging, curl'ing it
locally is good enough solution)

Re GPL:
it was the default choice, but I came to conclusion that for the
extra files from the wider radius, and which are also very light
on both the effective lines and the business value (given the
mentioned distance from the core), I am leaning towards explicit
"do whatever you like if you preserve the license/original
authorship notice".

I stuck with that license in coccinelle semantic patch, too,
which is of similar properties as "debugging helpers" (incl.
optional and unlikely downstream package delivery):
https://github.com/ClusterLabs/pacemaker/pull/1597
(that PR itself to be rebased, hence not referring the
particular commit)

Using relaxed licenses at places like these also makes it easier
to steer the license of the core, since files under these relaxed
licenses have no problem to be combined with practically anything,
by design.  So I'd like to encourage wider usage of either FSF
All Permissive license (or CC0) where it doesn't have any immediate
effect on the central parts and where stronger protections make
little sense.

Reply:
Ack. Please add a copy of the FSFAP to the licenses directory, and resubmit at least the FSFAP portions of this PR against the master branch (a new license is the sort of change that should be in master for a while so people can have a chance to evaluate it).